Applications/CommandLineInterface/ApplicationCommandLineInterface.cmake

```diff
@@ -24,18 +24,13 @@ set( ${APPLICATION}_INCLUDE_DIRS
 )
 set( ${APPLICATION}_SOURCE_FILES
   ${${APPLICATION}_SOURCE_DIR}/src/selxSuperElastix.cxx
-  ${${APPLICATION}_SOURCE_DIR}/src/selxConfigurationReader.cxx
 )
 
+// TODO SuperElastix Library should have an install, there should not be changing/configurable dependencies to subpart. 
 set( ${APPLICATION}_MODULE_DEPENDENCIES
```

Comment:
This is used for linking. How does it relate to install?

Reply:
yes, maybe I do not fully understand. 
My reasoning was: since the Command line application *is* our example how to embed the compiled superelastix library in an application, I would expect the build systems/processes to be quite separated. I can imagine the use case where people download our precompiled library and embed it in their tools (without having all objects from the libraries build directory). This downloaded (self-contained) set of files is the same what you should get if you 'install' the superelastix library. Maybe I'm wrong, but the module dependency system seems to be entangled with the libraries build output.
In my view the superElastix library install script knows what its dependencies are and can put that in the SuperElastix Use file, the application build system should not be bothered with that. 

Maybe we can start a discussion on this in a new issue?